zipline/gens/brokers/ib_broker.py

```diff
@@ -48,7 +48,17 @@
                                    'market_value', 'average_cost',
                                    'unrealized_pnl', 'realized_pnl',
                                    'account_name'])
-
+ExceptionSymbols = {}
```

Comment:
This code could be simplified by using two maps of defaultdict type:
```
symbol_to_exchange = defaultdict('SMART')
symbol_to_exchange['VIX'] = 'CBOE'
...
symbol_to_sec_type = defaultdict('STK')
symbol_to_sec_type['VIX'] = 'IND'
```

This way you would not need check_symbol() function.